lib/galaxy/tool_util/deps/container_classes.py

```diff
@@ -179,7 +179,7 @@ def add_var(name, value):
             if self.job_info.tool_directory:
                 defaults += ",$tool_directory:default_ro"
             if self.job_info.job_directory:
-                defaults += ",$job_directory:default_ro,$job_directory/outputs:rw"
+                defaults += ",$job_directory:default_ro,$job_directory/outputs:rw,$job_directory/configs"
```

Comment:
Should this end in ``:rw``?

Reply:
It should, I wonder why this seems to work regardless ?